src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/hmi/on_bc_system_capability_updated_notification_from_hmi_test.cc

```diff
@@ -47,15 +49,34 @@ using ::testing::Return;
 
 typedef std::shared_ptr<OnBCSystemCapabilityUpdatedNotificationFromHMI>
     OnBCSystemCapabilityUpdatedNotificationFromHMIPtr;
+typedef mobile_apis::SystemCapabilityType::eType SystemCapabilityType;
 
 namespace strings = application_manager::strings;
+
 namespace {
 const uint32_t kAppId = 1u;
 }  // namespace
 
 MATCHER(CheckMessageToMobile, "") {
   const auto function_id = mobile_apis::FunctionID::OnSystemCapabilityUpdatedID;
 
+  const bool is_function_id_matched =
+      function_id == static_cast<am::mobile_api::FunctionID::eType>(
+                         (*arg)[strings::params][strings::function_id].asInt());
+  const bool app_id_exist =
+      (*arg)[strings::msg_params].keyExists(strings::app_id);
+  bool is_connection_key_correct = true;
```

Comment:
make a lambda instead bool

Reply:
@LuxoftAKutsan Done in 6e05066361300834d09b5391052d19c305515d0e